Make sure cell pointers are aligned to pointer boundaries, for the sake of
strict-alignment 64-bit platforms.

--- pfe/def-config.h.orig	Sun May 11 12:29:19 2008
+++ pfe/def-config.h	Sun Jan 19 19:39:18 2014
@@ -127,7 +127,13 @@ typedef int mode_t;
 #endif
 
 #ifndef PFE_ALIGNOF_CELL
-#define PFE_ALIGNOF_CELL PFE_SIZEOF_INT
+# if defined PFE_SIZEOF_INT && PFE_SIZEOF_INT >= PFE_SIZEOF_VOIDP
+# define PFE_ALIGNOF_CELL PFE_SIZEOF_INT
+# elif defined PFE_SIZEOF_LONG && PFE_SIZEOF_LONG >= PFE_SIZEOF_VOIDP
+# define PFE_ALIGNOF_CELL PFE_SIZEOF_LONG
+# else
+# error cell type and size not detected.
+# endif
 #endif
 #ifndef PFE_ALIGNOF_SFLOAT
 #define PFE_ALIGNOF_SFLOAT PFE_SIZEOF_FLOAT
